libc/cmake/modules/LLVMLibCArchitectures.cmake

```
@@ -167,13 +167,13 @@ if(LIBC_TARGET_OS STREQUAL "baremetal")
   set(LIBC_TARGET_OS_IS_BAREMETAL TRUE)
 elseif(LIBC_TARGET_OS STREQUAL "linux")
   set(LIBC_TARGET_OS_IS_LINUX TRUE)
-elseif(LIBC_TARGET_OS STREQUAL "poky" OR LIBC_TARGET_OS STREQUAL "suse")
-  # poky are ustom Linux-base systems created by yocto. Since these are Linux
+elseif(LIBC_TARGET_OS STREQUAL "poky" OR LIBC_TARGET_OS STREQUAL "suse" OR LIBC_TARGET_OS STREQUAL "redhat")
```

```suggestion
elseif(LIBC_TARGET_OS STREQUAL "poky" OR LIBC_TARGET_OS STREQUAL "suse" OR
       LIBC_TARGET_OS STREQUAL "redhat")
```
We at least make an attempt to keep the CMake within 80 columns, sometimes unsuccessfully.

---

I tried to run `git clang-format` but it refused to format the file. I saw in `llvm/CMakeLists.txt` that some lines extended past 80 columns, so wasn't sure how to format the CMake file. I'll keep this in mind for the future.